lib/speaker/speakerFields.spec.js

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable no-console, max-len */
+
+import {
+  graphql,
+  GraphQLObjectType,
+  GraphQLSchema,
+} from 'graphql';
+
+import speakerFixtures from '../../test/fixtures/speakers.json';
+
+const mockfetch = {
+  fetchAllSpeakers: () => speakerFixtures.data.allSpeakers,
+  fetchSpeaker: () => speakerFixtures.data.allSpeakers[0],
+};
+
+const SpeakerFields = injectr('../../lib/speaker/speakerFields.js', {
+  '../fetch': mockfetch,
+});
+
+const SpeakerSchema = new GraphQLSchema({
+  query: new GraphQLObjectType({
+    name: 'Query',
+    fields: SpeakerFields.SpeakerSchemaFields,
+  }),
+});
+
+describe('Speaker Queries', () => {
+  it('should be able to get all available speakers', async () => {
+    const query = `
+    query {
+     	allSpeakers {
+     	  id
+        name
```

Comment:
Does eslint check inside strings? How would it know what format it's supposed to use?